src/debug.c

```diff
@@ -648,6 +657,22 @@ NULL
             ziplistRepr(o->ptr);
             addReplyStatus(c,"Ziplist structure printed on stdout");
         }
+    } else if (!strcasecmp(c->argv[1]->ptr,"quicklist") && (c->argc == 3 || c->argc == 4)) {
+        robj *o;
+
+        if ((o = objectCommandLookupOrReply(c,c->argv[2],shared.nokeyerr))
+            == NULL) return;
+
+        int full = 0;
+        if (c->argc == 4) {
+            full = atoi(c->argv[3]->ptr);;
```

Comment:
```suggestion
            full = atoi(c->argv[3]->ptr);
```
Btw, If there is only one line in the if, we don't need `{}`.